tests/test_skills/test_apy_estimation_abci/test_behaviours.py

```diff
@@ -818,17 +818,15 @@ def test_fetch_behaviour_non_indexed_block(
         else:
             res = {"errors": [{"message": "another error"}]}
 
-        response_kwargs["body"] = json.dumps(res).encode("utf-8")
-        behaviour.act_wrapper()
-        self.mock_http_request(request_kwargs, response_kwargs)
```

Comment:
The act which was triggering the logging was being called out of the `caplog`'s context manager.

Reply:
Yes, it is intentional. It was incorrect since I initially misplaced it, thinking that this was the call that was also triggering the logging.